test/test_utils/eks.py

```diff
@@ -316,12 +316,13 @@ def eks_setup(framework, cluster_name=None):
 
     kubectl_download_command = (
         f"curl --silent --location https://amazon-eks.s3-us-west-2.amazonaws.com/"
-        f"{EKS_VERSION}/2019-08-14/bin/{platform.lower()}/amd64/kubectl -o /tmp/kubectl"
+        f"{EKS_VERSION}/2019-08-14/bin/{platform.lower()}/amd64/kubectl -o /usr/local/bin/kubectl"
```

Comment:
Will it be 2019-08-22 or 2019-08-14 ?(https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html)

Reply:
For some reason, I was able to curl the binary on my laptop, so I didn't double-check the date. Thanks for pointing this out!